ISSUES.md

```diff
@@ -31,21 +31,12 @@ This is a list of known issues/TODOs for the project. Any fixes are welcomed.
         - Try to automate Windows SDK selection, and/or check properly for correct SDK and give clear error message
     - `main.cpp` - build fails with `DO_BA_BLOCK` and `DO_EIGEN`
 - Ceres
-    - Memory issue when building GMM at instance sizes above about d=20,K=50
+    - Memory issue when building GMM at instance sizes above about d=20, K=50
 - DiffSharp
-    - GMM build fails (expected float[] but got D[])
-    - BA is very slow (may just be slow, but could be an error)
-- Manual
-    - `ba_d.cpp` - refactor functions under `DO_CPP` to work without Eigen
-    - Perhaps add Hand derivative without Eigen
+    - GMM produces "NaN" for Jacobians when `d = 64`.
```

Comment:
I this still true? I thought the `logsumexp` patch was to fix that.

Reply:
No it isn't. I edited this file last time before fixing it. Updated.